lua/tests/infoview/lsp_spec.lua

```diff
@@ -34,7 +34,7 @@ describe('infoview', function()
     it('shows term state',
     function(_)
       local text = infoview_lsp_update({3, 23})
-      assert.has_all(text, {"⊢ ℕ"})
+      assert.has_all(text, {"▶ goals accomplished 🎉"})
```

Comment:
The result now means something completely different (previously 1 goal, now 0 goals).

Reply:
Ech, yes, will have to check what's happening here, I changed it too mindlessly.

Reply:
Fixed, this was me not properly checking that the goal response had any goals in it...